src/backends/webgl.ts

```diff
@@ -96,7 +137,7 @@ function get_prog_pair(ir: CompilerIR, progid: number) {
 
   // Get the fragment program.
   if (vertex_prog.quote_children.length > 1 ||
-      vertex_prog.quote_children.length < 1) {
+    vertex_prog.quote_children.length < 1) {
```

Comment:
Oh, I think the auto format of vscode automatically added some whitespaces here. Maybe I should carefully check what I commit next time.